ghcide/ghcide.cabal

```
@@ -258,7 +259,8 @@ executable ghcide
                 -rtsopts
                 -- disable idle GC
                 -- increase nursery size
-                "-with-rtsopts=-I0 -A128M"
+                -- Enable collection of heap statistics
+                "-with-rtsopts="-I0 -A128M -T"
```

I think you have extraneous quote here. I recently struggled with this too, see [this](https://blog.clement.delafargue.name/posts/2017-07-21-multiple-rts-options.html)
```suggestion
                "-with-rtsopts=-I0 -A128M -T"
```